src/Main.tsx

```diff
@@ -54,7 +51,7 @@ export class Container extends React.Component<Properties> {
   render() {
     const mainClassName = classNames('main', {
       'context-panel-open': this.props.isContextPanelOpen,
-      'sidekick-panel-open': this.props.isSidekickOpen && this.props.context.isAuthenticated,
+      'sidekick-panel-open': this.props.context.isAuthenticated,
```

Comment:
As a note: `public ZOS` is also not a thing any more so we shouldn't even need to add this class. It should just be that way all the time. `has-context-panel` can also be removed because its state should also be constant now.

Doesn't mean you have to do it. Just noting more potential cleanup.

Reply:
got it